PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -513,7 +513,7 @@ public function prepare_content($channel, $class_id, $weight, $body_size, $packe
      * @param int $channel
      * @param array $method_sig
      * @param AMQPWriter|string $args
-     * @param null $pkt
+     * @param AMQPWriter|null $pkt
```

Comment:
This part should be merged as PHPStan correctly calls this invalid as just null.